styles/helpers/menu-slide.css

```diff
@@ -42,6 +42,26 @@ title: Menu Slide
 </div>
 <div class="col-11"><h1>ClientKit</h1></div>
 ```
+
+### Mobile Only
+
+```example:html
+<div class="col-9"><h1>ClientKit</h1></div>
+<div class="col-3">
+  <div class="menu-slide mobile-only">
```

Comment:
I don't like the `mobile-only` class. It's confusing because the stuff underneath still shows up on non-mobile